app/assets/javascripts/pageflow/editor/views/theme_view.js

```diff
@@ -0,0 +1,67 @@
+/** @api private */
+pageflow.ThemeView = Backbone.Marionette.ItemView.extend({
+  template: 'pageflow/editor/templates/theme',
+  tagName: 'li',
+
+  ui: {
+    name: '.theme_name',
+    selectButton: '.select_theme',
+    selectedRegion: '.theme_selected'
+  },
+
+  events: {
+    'click .select_theme': function(event) {
+      this.entry.configuration.set('theme_name', this.model.get('name'));
+      pageflow.updateStylesheet('theme');
+    },
+    'mouseenter': function() {
+      var that = this;
+      this.uiPreviewImage.attr('src', pageflow.editor.themes.find(function(theme) {
+        return theme.attributes.name == that.model.get('name');
+      }).get('preview_image_url'));
+      this.uiPreviewHeaderThemeName.text(that.translateThemeName());
+    },
+    'mouseleave': function() {
+      var that = this;
+      this.uiPreviewImage.attr('src', pageflow.editor.themes.find(function(theme) {
+        return theme.attributes.name == pageflow.entry.configuration.attributes.theme_name;
+      }).get('preview_image_url'));
+      this.uiPreviewHeaderThemeName.text(that.translateThemeName({
+        name: pageflow.entry.configuration.attributes.theme_name
+      }));
+    }
```

Comment:
I agree. Adding to this line of thought, I think it is more appropriate in this case to even use a `preview` Backbone.Model in addition to a `selection` Backbone.Model, since previewed theme and selected theme can differ. This implies reusing `selectableView` and writing a very similar `previewableView`.